WorldBank-Transport/DRIVER
# Add translation file for French language

## Overview

Adds French language translations to the web app static text.

### Checklist




### Notes

Left exclaim text where I didn't have translations

## Testing Instructions

 * Load the app and make sure you have French selected in your group_vars file (I think).
 * Use the language picker to select French
